src/framework/ui/view/iconcodes.h

```diff
@@ -201,7 +201,7 @@ class IconCode
         CLEF_TREBLE = 0xF31A,
 
         AUTO_TEXT = 0xF329,
-        NOTE_HEAD_EIGHTH = 0xF33A,
+        NOTE_HEAD_8TH = 0xF33A,
```

Comment:
I just realised that this icon is specifically designed to match the other beaming icons, so maybe we should call it `BEAM_NONE` to make that clear.